test/functional/apps/visualize/_shared_item.js

```diff
@@ -41,6 +41,8 @@ export default function ({ getService, getPageObjects }) {
         const { title, description } = await PageObjects.common.getSharedItemTitleAndDescription();
         expect(title).to.eql(expected.title);
         expect(description).to.eql(expected.description);
+        const sharedItemContainer = await PageObjects.common.getSharedItemContainer();
+        expect(sharedItemContainer).to.defined;
```

Comment:
should this be `to.be.defined`? Also, can we check node length instead of just `defined`?

Reply:
Oh, right, thanks! 
that line 45 didn't make any sense, as anyway assertion happened within getSharedItemContainer call. 

I also changed to checking length as you suggested 👍